cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/service/http/HttpHandlerGeneratorTest.java

```diff
@@ -570,12 +570,23 @@ public RunId getRunId() {
 
     @Override
     public <T extends Dataset> T getDataset(String name) throws DatasetInstantiationException {
-      return getDataset(name, null);
+      return getDataset(null, name);
```

Comment:
is namespace @Nullable in DatasetContext? If not, then you should not pass in null.